src/views/CardStackTransitioner.js

```diff
@@ -0,0 +1,181 @@
+     * card, and will animate in together.
+     *
+     * The default for `modal` mode is `screen`, and the default for `card` mode
+     * is `screen` on Android and `float` on iOS.
+     */
+    headerMode: PropTypes.oneOf(['float', 'screen', 'none']),
+
+    /**
+     * Custom React component to be used as a header
+     */
+    headerComponent: PropTypes.func,
+
+    /**
+     * Style of the cards movement. Value could be `card` or `modal`.
+     * Default value is `card`.
+     */
+    mode: PropTypes.oneOf(['card', 'modal']),
+
+    /**
+     * Optional custom animation when transitioning between screens.
+     */
+    transitionConfig: PropTypes.func,
+
+    /**
+     * The navigation prop, including the state and the dispatcher for the back
+     * action. The dispatcher must handle the back action
+     * ({ type: NavigationActions.BACK }), and the navigation state has this shape:
+     *
+     * ```js
+     * navigation.state = {
+     *   index: 0, // the index of the selected route.
+     *   routes: [ // A list of routes.
+     *     {key: 'page 1'}, // The 1st route.
+     *     {key: 'page 2'}, // The second route.
+     *   ],
+     * };
+     * ```
+     */
+    navigation: NavigationPropTypes.navigation.isRequired,
+
+    /**
+     * Custom style applied to the cards stack.
+     */
+    style: View.propTypes.style,
+  };
+
+  static defaultProps: DefaultProps = {
+    mode: 'card',
+    headerComponent: Header,
+  };
+
+  render() {
+    return (
+      <Transitioner
+        configureTransition={this._configureTransition}
+        navigation={this.props.navigation}
+        render={this._render}
+        style={this.props.style}
+        onTransitionStart={this.props.onTransitionStart}
+        onTransitionEnd={this.props.onTransitionEnd}
+      />
+    );
+  }
+
+  _configureTransition = (
+    // props for the new screen
+    transitionProps: NavigationTransitionProps,
+    // props for the old screen
+    prevTransitionProps: NavigationTransitionProps
+  ) => {
+    const isModal = this.props.mode === 'modal';
+    // Copy the object so we can assign useNativeDriver below
+    // (avoid Flow error, transitionSpec is of type NavigationTransitionSpec).
+    const transitionSpec = {
+      ...TransitionConfigs.getTransitionConfig(
+        this.props.transitionConfig,
+        transitionProps,
+        prevTransitionProps,
+        isModal,
+      ).transitionSpec,
+    };
+    if (
+      !!NativeAnimatedModule &&
+      // Native animation support also depends on the transforms used:
+      CardStackStyleInterpolator.canUseNativeDriver(isModal)
+    ) {
+      // Internal undocumented prop
+      transitionSpec.useNativeDriver = true;
+    }
+    return transitionSpec;
+  };
+
+  _render = (props: NavigationTransitionProps): React.Element<*> => {
+    const {
+      headerComponent,
+      headerMode,
+      mode,
+      router,
+      style,
+    } = this.props;
+    return (
+      <CardStack
```

Comment:
As per this http://facebook.github.io/react-native/docs/navigation.html, CardStack is deprecated as of React native 0.43